src/components/media_manager/src/media_manager_impl.cc

```diff
@@ -313,7 +313,7 @@ void MediaManagerImpl::OnMessageReceived(
   }
 
   if (!application_manager_.CanAppStream(streaming_app_id, service_type)) {
-    application_manager_.ForbidStreaming(streaming_app_id);
+    application_manager_.ForbidStreaming(streaming_app_id, service_type);
     LOG4CXX_ERROR(logger_,
                   "The application trying to stream when it should not.");
```

Comment:
@SNiukalov could you pls update this message and include service_type here as well?

Reply:
@AKalinich-Luxoft 
Thanks, fixed in:https://github.com/smartdevicelink/sdl_core/pull/3293/commits/cde8652542fc0a0e431ae0a73e877022041d35da